xml/System.IO/MatchType.xml

```diff
@@ -42,7 +42,7 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Match using '*' and '?' wildcards.</summary>
```

Comment:
"Match using simple '*' and '?' wildcards."

remarks:

```
"'*' matches zero to any amount of characters. '?' matches exactly one character. '*.*' matches any name with a period in it (with `Win32` this would match all items)"
```

Reply:
This is the recommended option for matching. Performance is better and the rules are easier to understand.

Reply:
Thanks. Added.